bridges/relays/utils/src/metrics.rs

```diff
@@ -121,9 +121,9 @@ impl From<Option<MetricsAddress>> for MetricsParams {
 /// Returns metric name optionally prefixed with given prefix.
 pub fn metric_name(prefix: Option<&str>, name: &str) -> String {
 	if let Some(prefix) = prefix {
-		format!("{}_{}", prefix, name)
+		format!("polkadot_{}_{}", prefix, name)
 	} else {
-		name.into()
+		format!("polkadot_{}", name)
```

Comment:
This piece of code isn't Polkadot specific (and in fact, isn't technically used by the Polkadot repo), so there's no need for this change